### PR TITLE
UP-4853: Convert admin tools portlet to app launchers

### PR DESCRIPTION
**Pull request by ChristianMurphy**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [x] view conforms with [WCAG 2.0 AA][]

##### Description of change

<https://issues.jasig.org/browse/UP-4853>

In the spirit of continuing to improve uPortals user interface @drewwills  suggested that the admin tools portlet, which is currently a list of plain text links, could be converted into a series of app launchers. One launcher for each link with icons making each one easier to identify and making the admin page easier on the eyes.

screencapture-localhost-8080-uportal-f-admin-normal-render-up-1494884115321

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

**Comment by drewwills** [comment]
Some requests...

  1. Add app-launchers for Announcements Admin, Please Register, and Emergency Alert Admin (and remove those 3 from the layout)
  1. Publish all with `chrome=no-chrome`
  1. Re-do the icons with some color and perhaps a touch of drop shadow... something to give them some texture

**Comment by bjagg** [comment]
Nice visual improvement!

**Comment by cousquer** [comment]
@ChristianMurphy 👍 
Definitely a very nice idea to use app-launcher for that ! for what told you @drewwills you can find also insight in [Ionicons](http://ionicons.com/) (MIT License).
It could be perhaps great to find for uPortal 5 or 6 a way to centralized the place where artworks (img, font, svg) are. Because pictures (drawer img, app-launcher icons, mobile icons, etc.) are locate at different places in tree.
something like https://myhuskynet.univ.edu/artworks/images  https://myhuskynet.univ.edu/artworks/font
I don't know if it's something interesting but it may be practical.

**Comment by ChristianMurphy** [comment]
> 1. Add app-launchers for Announcements Admin, Please Register, and Emergency Alert Admin (and remove those 3 from the layout)
> 2. Publish all with `chrome=no-chrome`

Done and Done!
Thanks for the suggestions

screencapture-localhost-8080-uportal-f-admin-normal-render-up-1494970648407

> 3. Re-do the icons with some color and perhaps a touch of drop shadow... something to give them some texture

I agree this could use some more spice. :hot_pepper: 
However with the current implementation of app launchers, adding color or shadow effects would require modifying the original font-awesome images.
Those images fall under the [Open Font License](http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL) which has a share and share alike cause that would require uPortal also be under the Open Font License. :scroll: 
An approach that could preserve license integrity and address @cousquer's comment

> It could be perhaps great to find for uPortal 5 or 6 a way to centralized the place where artworks (img, font, svg) are

Would be to add icon font support to app launchers, so the fonts could be styled with CSS which is allowed under the Open Font License, as well as allowing for centralization in Resource Server.

**Comment by cousquer** [comment]
👍🏻 That's exactly what I was thinking, christian.

**Comment by cousquer** [comment]
Jaffathecake has write something really interesting for you:
https://jakearchibald.com/2017/combining-fonts/

**Comment by drewwills** [comment]
@ChristianMurphy -- I've glanced through all the XML deltas and they look great.  :+1: 

Do you have time to export new versions of the images with some transparent padding around the icons, like we brainstormed?  (I think those would look better.)

File: uportal-war/src/main/data/quickstart_entities/portlet-definition/user-administration-launcher.portlet-definition.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<portlet-definition version="4.3" xsi:schemaLocation="https://source.jasig.org/schemas/uportal/io/portlet-definition https://source.jasig.org/schemas/uportal/io/portlet-definition/portlet-definition-4.3.xsd" xmlns="https://source.jasig.org/schemas/uportal/io/portlet-definition" xmlns:ns6="https://source.jasig.org/schemas/uportal/io/portlet-type" xmlns:ns5="https://source.jasig.org/schemas/uportal/io/stylesheet-descriptor" xmlns:ns8="https://source.jasig.org/schemas/uportal/io/event-aggregation" xmlns:ns7="https://source.jasig.org/schemas/uportal/io/subscribed-fragment" xmlns:ns2="https://source.jasig.org/schemas/uportal" xmlns:ns4="https://source.jasig.org/schemas/uportal/io/user" xmlns:ns3="https://source.jasig.org/schemas/uportal/io/permission-owner" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <title>User Administration Launcher</title>
+    <name>UUser Administration Launcher</name>
```
Comment (apetro) [inline uportal-war/src/main/data/quickstart_entities/portlet-definition/user-administration-launcher.portlet-definition.xml]: "UU"

Comment (ChristianMurphy) [inline uportal-war/src/main/data/quickstart_entities/portlet-definition/user-administration-launcher.portlet-definition.xml]: Good catch! Updated

**Comment by drewwills** [comment]
@apetro -- Looks like your feedback was implemented.  I'm going to go ahead and merge.

**Pull request merged**